client/lib/products-values/constants.js

```diff
@@ -90,47 +90,48 @@ export const getJetpackProductsShortNames = () => {
 };
 
 export const getJetpackProductsDisplayNames = () => {
+	const backupDaily = (
+		<>
+			{ translate( 'Jetpack Backup {{em}}Daily{{/em}}', {
+				components: {
+					em: <em />,
+				},
+			} ) }{ ' ' }
+		</>
+	);
+
+	const backupRealtime = (
+		<>
+			{ ' ' }
+			{ translate( 'Jetpack Backup {{em}}Real-Time{{/em}}', {
+				components: {
+					em: <em />,
+				},
+			} ) }{ ' ' }
+		</>
+	);
+
+	const search = translate( 'Jetpack Search' );
+
+	const scanDaily = (
+		<>
+			{ translate( 'Jetpack Scan {{em}}Daily{{/em}}', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **33** times:
`translate( 'Jetpack Backup {{em}}Daily{{/em}}' )` *ES Score: 7*